app/models/area.rb

```diff
@@ -3,4 +3,9 @@ def self.get(area_code)
     attributes = Etsource::Loader.instance.area_attributes(area_code)
     attributes.with_indifferent_access
   end
+
+  def self.derived?(area_code)
+    Atlas::Dataset.exists?(area_code) &&
+      get(area_code).fetch(:derived)
```

Comment:
Looks like this is short enough to put on one line without exceeding 80 characters?